### PR TITLE
Update install namespace and create manged clusters for KUTTL tests

### PR DESCRIPTION
**Pull request by irinamihai**

### CODE REVIEW

File: controllers/clustergroupupgrade_controller.go [hunk 1]
```diff
@@ -1043,11 +1043,12 @@ func (r *ClusterGroupUpgradeReconciler) validateCR(ctx context.Context, clusterG
...
 		})
 		err := r.Client.Get(ctx, client.ObjectKey{
-			Name: cluster,
+			Name:      cluster,
+			Namespace: cluster,
 		}, foundManagedCluster)
 
 		if err != nil {
-			return fmt.Errorf("Cluster %s is not a ManagedCluster", cluster)
+			return fmt.Errorf("Cluster %s is not a ManagedCluster!!!", cluster)
```
Comment (irinamihai) [inline controllers/clustergroupupgrade_controller.go:1051]: Remove this.

Comment (rcarrillocruz) [reply controllers/clustergroupupgrade_controller.go:1051]: It seems you are adding exclamations, yet you self-commented to remove them? Is it 'reminder' you put yourself to address it shortly?

File: deploy/upgrades/canaries/clustergroupupgrade.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: ran.openshift.io/v1alpha1
 kind: ClusterGroupUpgrade
 metadata:
   name: canaries
-  namespace: default
+  namespace: openshift-cluster-group-upgrades
```
Comment (rcarrillocruz) [inline deploy/upgrades/canaries/clustergroupupgrade.yaml:5]: Why you change all the namespaces of the CRs for testing?
The 'openshift-cluster-group-upgrades' namespace is supposedly where the controller will be placed unto, but the user can and should be able to open ClusterGroupUpgrade CRs on any namespace.

File: controllers/templates.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ spec:
             exclude:
               - kube-*
             include:
-              - default
+              - openshift-cluster-group-upgrades
```
Comment (rcarrillocruz) [inline controllers/templates.go:71]: you don't need to add this. THe namespace openshift-cluster-group-upgrades is a namespace where the controller runs, which is on the hub. This namespace selector field is on configuration policies, which are meant on the spokes.
As a matter of fact, this 'default' is a bug, I should probably change to * .

**Pull request merged**